src/SFA.DAS.EAS.Web/ViewModels/CreateCommitmentViewModel.cs

```diff
@@ -23,9 +23,16 @@ public sealed class SelectTransferConnectionViewModel
         [Required(ErrorMessage = "Choose an option")]
         public string TransferConnectionCode { get; set; }
 
-        public IEnumerable<TransferConnection> TransferConnections { get; set; }
+        public IEnumerable<TransferConnectionViewModel> TransferConnections { get; set; }
     }
 
+    public sealed class TransferConnectionViewModel
```

Comment:
This could be extracted into its own file (and so could the other classes in the file).

Reply:
No for some reason all the classes are in this one file

Reply:
If all the classes have a logical grouping, then the filename should represent that, rather than having the name of a single class in the file.